### ISSUE TITLE
Genome assembly

### ISSUE THREAD

**Issue by ValentinaBoP** [mcstitzer/maize_genomic_ecosystem#1]
Hi!

I'm not entirely sure about which genome assembly file is used in the code. In the GenomeInfo.R, it says:

```
## names of the genome and its short ID assigned by the maizegdb nomenclature committee (https://www.maizegdb.org/nomenclature)
GENOME='B73'
SHORTID='Zm00001d'

## the path to the genome fasta (including the filename)
GENOMEFA='/home/mstitzer/projects/agpv4_te_annotation/ncbi_pseudomolecule/B73V4.both_pseudo_AND_unplaced.fa'
```

Is it referring to this? https://ftp.ncbi.nlm.nih.gov/genomes/all/GCA/000/005/005/GCA_000005005.6_B73_RefGen_v4/
If so, do the sequence names correspond to the version you have used?

Thanks!
Valentina

**Comment by mcstitzer** [comment]
Hi Valentina,

The GenBank record you posted is the same assembly version (B73v4), but NCBI assigns names to chromosomes differently. For example, CM007647.1 is chromosome 1. Conversions in the table at the bottom here:
https://www.ncbi.nlm.nih.gov/assembly/GCF_000005005.2/

My chromosome and unplaced scaffold names closely match the maizegdb fasta available here: 
https://download.maizegdb.org/Zm-B73-REFERENCE-GRAMENE-4.0/Zm-B73-REFERENCE-GRAMENE-4.0.fa.gz

The minor difference is that chromosomes are named Chr1-Chr10, while in the file I used, chromosomes were named 1-10. Unplaced scaffolds are named identically in both files, e.g. B73V4_ctg237. One other difference is that the maizegdb file includes a fasta record for the chloroplast genome (named Pt), while the fasta I used did not include plastids.

Apologies that there are likely small differences in chromosome sequence names throughout files in this repository, like Chr1 vs chr1 vs 1!

Best,
Michelle
